tests/server/blueprints/cases/test_cases_views.py

```diff
@@ -14,46 +11,60 @@
 TEST_TOKEN = "test_token"
 
 
-@pytest.mark.parametrize("report_types", list(CUSTOM_CASE_REPORTS.keys()))
-def test_custom_report(app, institute_obj, case_obj, report_types):
+def test_custom_report(app, institute_obj, case_obj, tmp_path):
```

Comment:
To make sure that this test is good I've tried reverting the code in cases_views and indeed the error is caught:

<img width="777" alt="image" src="https://user-images.githubusercontent.com/28093618/227179046-c4412fc5-8b4e-469f-a92e-5e54e2cd63b9.png">

The tests fails with this message:

<img width="1035" alt="image" src="https://user-images.githubusercontent.com/28093618/227179124-fd65359d-01a9-492f-9b83-c22e81b32841.png">

That happens when downloading the PDF report for the delivery reports -> instead of the PDF report you get an error page